BringYourOwnBoard.md

```diff
@@ -0,0 +1,126 @@
+
+### What do I need to think about? 
+
+Whatever system you're doing you need a way to make the A, B, and adu partitions. You can use `wic` like we do or if the tooling from your BSP doesn't support / won't work with `wic` they will provide another method for accomplishing the same task. 
+
+As for making your image you can peruse what we put in our image (look at the `IMAGE_INSTALL` section in the [adu-base-image file](./recipes-core/images/adu-base-image.bb) ), but the most important thing is to include the following assuming you are making no changes to `meta-azure-device-update`: 
+
+1. adu-swupdate-hw-compat
+2. adu-device-info-files
+3. adu-agent-service
+
+If you have questions about what these are and what they do in the actual image please read the doc in [meta-azure-device-update here]( https://github.com/Azure/meta-azure-device-update). 
+
+## recipes-extended
+
+### What does it do? 
+
+The [recipes-extended](./recipes-extended/) section of the repository is the tooling for generating the SwUpdate file (the `*.swu`) that allows SwUpdate to install the image. Each `*.swu` file must contain a [sw-description file](./recipes-extended/images/adu-update-image/raspberrypi4-64/sw-description) which describes the goal state of the device. You can read more about what a sw-description file is, options you can use, and how to write your own [here](https://sbabic.github.io/swupdate/sw-description.html). 
+
+As apart of the `*.swu` generation swupdate will require that you sign the file with a private key so that the public key that is installed on the device at build time can be used to verify the image. You can read more about how we generate our keys for the proof of concept [here](https://github.com/Azure/iot-hub-device-update-yocto/blob/main/keys/README.md) as well as some recommendations for what to look for in production. 
+
+### What do I need to think about? 
+
+For a customer looking at porting this section over there's not a lot you need to chagne. You need to edit the file such that it depends on your own custom image's bitbake file, edit the sw-description file to reflect what you would like the system to look like after an update, and provide tooling for accessing a private signing key. 
```

Comment:
Typo: _For a customer looking at porting this section over there's not a lot you need to **chagne**_